src/jshal.js

```diff
@@ -118,20 +118,20 @@ mergeInto(LibraryManager.library, {
   },
 
   mp_js_hal_accelerometer_get_x: function () {
-    return board.accelerometer.x.value;
+    return board.accelerometer.state.accelerometerX.value;
```

Comment:
Likewise quite messy

Reply:
Living with this for now. It's not API and it does make it easier to implement setValue.